### PR TITLE
introduce variable to support thresholding during inference

### PR DESCRIPTION
**Pull request by ramchandra-karandikar**
Creating draft PR. Need to test TensorRT generation with the new configuration file

### CODE REVIEW

File: configs/centerpoint/centerpoint_03pillar_kitti_lum.py [hunk 1]
```diff
@@ -267,6 +267,7 @@
 output_scatter_shape = [552, 640]
 new_outsize_factor = 4
 pillar_feature_in_channels_new = 4
+cpp_inference_threshold_lum = 0.3
```
Comment (siddharthanpr) [inline configs/centerpoint/centerpoint_03pillar_kitti_lum.py:270]: call this score_threshold_cpp_inference_lum

Comment (siddharthanpr) [inline configs/centerpoint/centerpoint_03pillar_kitti_lum.py:270]:
```suggestion
score_threshold_cpp_inference_lum = 0.3
```

File: mmdet3d/apis/trt_build.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 additional_outputs = dict(
     voxel_size=centerpoint_config.voxel_size_lum,
+    cpp_inference_threshold_lum=centerpoint_config.cpp_inference_threshold_lum,
```
Comment (siddharthanpr) [inline mmdet3d/apis/trt_build.py:16]:
```suggestion
   score_threshold=centerpoint_config.score_threshold_cpp_inference_lum,
```

File: configs/centerpoint/centerpoint_03pillar_kitti_lum.py [hunk 3]
```diff
@@ -267,6 +267,7 @@
...
 model = dict(
     type='CenterPoint',
     pts_voxel_layer=dict(
```
Comment (siddharthanpr) [inline configs/centerpoint/centerpoint_03pillar_kitti_lum.py:273]: Disclaimer Not related to the PR: we have inconsistent values here ```
voxel_size=[0.2, 0.2, 16],
        max_voxels=(30000, 40000),
        point_cloud_range=[0, -55.2, -15, 128, 55.2, 15]),

Comment (siddharthanpr) [reply configs/centerpoint/centerpoint_03pillar_kitti_lum.py:273]: can we llook into this in future PR?

**Review by siddharthanpr — CHANGES_REQUESTED** [review]
small changes

File: configs/centerpoint/centerpoint_03pillar_kitti_lum.py (hunk 3 above)
Comment (ramchandra-karandikar) [reply configs/centerpoint/centerpoint_03pillar_kitti_lum.py:273]: Yes, I will create a separate PR for those. 
Need to clean-up unused variables

File: configs/centerpoint/centerpoint_03pillar_kitti_lum.py [hunk 4]
```diff
@@ -380,6 +378,3 @@
 mp_start_method = 'fork'
 total_epochs = 210
 gpu_ids = range(0, 8)
-conversion = dict(
```
Comment (siddharthanpr) [inline configs/centerpoint/centerpoint_03pillar_kitti_lum.py:383]: i think this is needed for engine conversion

File: mmdet3d/apis/trt_build.py [hunk 5]
```diff
@@ -13,6 +13,7 @@
 
 additional_outputs = dict(
     voxel_size=centerpoint_config.voxel_size_lum,
+   score_threshold=centerpoint_config.score_threshold_cpp_inference_lum,
```
Comment (siddharthanpr) [inline mmdet3d/apis/trt_build.py:16]: can you format this.

**Comment by siddharthanpr** [comment]
closing this since we figured out that -8, 8 is not needed

**Pull request closed without merging**